gm4_mob_curing/data/gm4_mob_curing/functions/cure/mooshroom.mcfunction

```diff
@@ -1,8 +1,9 @@
-#@s = cows with slowness needing conversion at @s
-#run from main
+# @s = cows with slowness needing conversion at @s
+# run from main
+
 summon mooshroom
```

Comment:
This freshly summoned mob will face south by default, which will look slightly disappointing visually. Try to teleport your new mob to the old one to prevent this.

Also, I'm not sure how vanilla's curing handles NBT, but do we want to transfer things like `CustomName`?

_This comment also applies to the other curing flavors._

Reply:
Comparing against lightning and pig->piglin, it summons a new entity.  This copies that functionality, though I'll take the suggestion of teleporting in-place and copying all data, to match the way Shroomites does it.